gen/dcos-config-windows.yaml

```diff
@@ -1,20 +1,4 @@
 package:
-  - path: c:\pkginfo.json
-    content: |
-      {
-        "environment": {
-          "MESOS_IP_DISCOVERY_COMMAND": "{{ windows_dcos_install_path_json }}}\\bin\\detect_ip.ps1",
-          "PROVIDER": "{{ provider }}"
-        }
-      }
```

Comment:
@KostiantynOleksiienko please keep that section as a quick fix.
Please update the location of this file to:    
```
- path: {{ windows_dcos_install_path }} \etc\pkginfo.json
```
so that on the final windows node the file is located here: C:\d2iq\dcos\etc\pkginfo.json

Reply:
Ok, I will do it in 10 minutes.